### PR TITLE
Remove test progress statuses on reset

### PR DESCRIPTION
**Pull request by lukastymo**

### CODE REVIEW

File: app/repositories/application/OnlineTestRepository.scala [hunk 1]
```diff
@@ -184,6 +186,21 @@ class OnlineTestMongoRepository(dateTime: DateTimeFactory)(implicit mongo: () =>
     collection.update(query, applicationStatusBSON, upsert = false) map ( _ => () )
   }
 
+  override def removeProgressStatuses(appId: String, progressStatuses: List[ProgressStatus]): Future[Unit] = {
```
Comment (benjstephenson) [inline app/repositories/application/OnlineTestRepository.scala]: If this method only works for PHASE1 statuses then I think we should make this clear in the method name.

File: app/services/onlinetesting/OnlineTestService.scala [hunk 2]
```diff
@@ -112,6 +111,17 @@ trait OnlineTestService {
     registerAndInviteForTestGroup(application, getScheduleNamesForApplication(application))
   }
 
+  def resetPhase1Tests(application: OnlineTestApplication, testNamesToRemove: List[String]): Future[Unit] = {
+    for {
+    - <- registerAndInviteForTestGroup(application, testNamesToRemove)
+    } yield {
+      auditService.logEventNoRequest(
+        "Phase1TestsReseted",
```
Comment (benjstephenson) [inline app/services/onlinetesting/OnlineTestService.scala]: "Phase1TestsReset"

File: app/repositories/application/OnlineTestRepository.scala (hunk 1 above)
Comment (lukastymo) [inline app/repositories/application/OnlineTestRepository.scala]: I thought that the whole repo is for Phase1 (bad naming). I like small repos. We have already few methods only for phase1.

Comment (lukastymo) [inline app/repositories/application/OnlineTestRepository.scala]: Anyway it's worth to keep same naming. I'll renam to phase1

**Pull request merged**